decidim-core/config/locales/en.yml

```diff
@@ -1536,6 +1536,15 @@ en:
         reason: Reason
         report_html: <p>The following <a href="%{url}">content</a> has been hidden automatically.</p>
         subject: A resource has been hidden automatically
+      hidden_by_admin:
+        content: Reported content
+        details: Details
+        hello: Hello %{name},
+        manage_moderations: Manage moderations
+        participatory_space: Participatory space
+        reason: Reason
+        report_html: <p>The following <a href="%{url}">content</a> has been hidden by %{admin}.</p>
```

Comment:
```suggestion
        report_html: <p>The following <a href="%{url}">content</a> has been hidden by %{moderator}.</p>
```

Reply:
Suggestion applied in 8555f7f9dd . Done without Github's help as we needed to rename a file.